tests/test_transformers/test_sqltransform.py

```diff
@@ -17,44 +26,30 @@
 DUMMY_CLASS = "c"
 
 
-"""
-Tests transformation from a linkml model to a relational model (independent of SQL).
-
-Note: This tests the transformation between one LinkML model and another.
-
-The input model may include mulitvalued fields, but these are transformed away in
-the relational representation.
-"""
-
-
 def _translate(builder: SchemaBuilder) -> TransformationResult:
     sv = SchemaView(builder.schema)
     sqltr = RelationalModelTransformer(sv)
     return sqltr.transform()
 
 
-def test_inject_primary_key():
-    """
-    test a minimal schema with no primary names declared
-    """
+def test_inject_primary_key() -> None:
+    """Test a minimal schema with no primary names declared."""
     b = SchemaBuilder()
     slots = ["name", "description"]
     b.add_class(DUMMY_CLASS, slots)
     rel_schema = _translate(b).schema
-    assert Counter(slots + ["id"]) == Counter(rel_schema.classes[DUMMY_CLASS].attributes.keys())
+    assert set(rel_schema.classes[DUMMY_CLASS].attributes.keys()) == {*slots, "id"}
```

Comment:
more accurate to test what the attributes are than just to test how many there are